src/lib/libhtml5.js

```diff
@@ -806,6 +807,52 @@ var LibraryHTML5 = {
   emscripten_set_focusout_callback_on_thread: (target, userData, useCapture, callbackfunc, targetThread) =>
     registerFocusEventCallback(target, userData, useCapture, callbackfunc, {{{ cDefs.EMSCRIPTEN_EVENT_FOCUSOUT }}}, "focusout", targetThread),
 
+  $registerInputEventCallback__proxy: 'sync',
+  $registerInputEventCallback__deps: ['$JSEvents', '$findEventTarget', '$stringToUTF8', '$lengthBytesUTF8', 'malloc', 'realloc'],
+  $registerInputEventCallback: (target, userData, useCapture, callbackfunc, eventTypeId, eventTypeString, targetThread) => {
+#if PTHREADS
+    targetThread = JSEvents.getTargetThreadForEventCallback(targetThread);
+#endif
+
+    var inputEventHandlerFunc = (e = event) => {
+      var data = e.data || '';
+      var inputType = e.inputType || '';
+      var isComposing = e.isComposing;
+      const dataLengthWithTermination = lengthBytesUTF8(data) + 1;
+      const eventSize = {{{ C_STRUCTS.EmscriptenInputEvent.__size__ }}} + dataLengthWithTermination;
```

Comment:
We don't use `const` yet in most of the system JS libraries.  Can you just use `var` here?